pages/mesosphere/dcos/services/edge-lb/1.5/getting-started/edge-lb-mlb-example/index.md

```diff
@@ -0,0 +1,240 @@
+---
```

Comment:
Does it make sense to merge this with the 'Comparing load balancers' page?

Reply:
Creating separate Jira ticket.

Reply:
Created a separate ticket.